src/shared/components/mutationTable/column/cancerCellFraction/CancerCellFractionElement.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import SampleManager from 'pages/patientView/SampleManager';
+
+export const maxBarHeight = 12;
```

Comment:
Do we use `maxBarHeight ` out of file?

Reply:
using it in the unit tests to make sure we generate a bar with total height equaling maxBarHeight!

Reply:
@zhx828

Reply:
Ah ok, now I see it. The file was folded. Thanks!

Reply:
I think this should be all caps:
```suggestion
export const MAX_BAR_HEIGHT = 12;
```

Reply:
@zhx828 Yes it is used in the `CancerCellFractionElement.spec.tsx` file for unit testing.